app/src/main/java/com/cornellappdev/uplift/ui/screens/CapacityReminderScreen.kt

```diff
@@ -0,0 +1,112 @@
+package com.cornellappdev.uplift.ui.screens
+
+import android.annotation.SuppressLint
+import androidx.compose.animation.AnimatedVisibility
+import androidx.compose.animation.fadeIn
+import androidx.compose.animation.fadeOut
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.cornellappdev.uplift.ui.components.capacityreminder.CapacityReminderSwitch
+import com.cornellappdev.uplift.ui.components.capacityreminder.CapacityThreshold
+import com.cornellappdev.uplift.ui.components.general.UpliftTopBarWithBack
+import com.cornellappdev.uplift.util.GRAY03
+import com.cornellappdev.uplift.util.montserratFamily
+import com.cornellappdev.uplift.ui.components.capacityreminder.LocationsToRemind
+import com.cornellappdev.uplift.ui.components.capacityreminder.ReminderDays
+
+/**
+ * Screen for the capacity reminder feature.
+ */
+@SuppressLint("UnusedMaterial3ScaffoldPaddingParameter")
+@Composable
+fun CapacityReminderScreen(
+    checked: Boolean,
+    onChecked: (Boolean) -> Unit = {},
+    initialSelectedDays: Set<String> = emptySet(),
+    onDaySelected: (Set<String>) -> Unit = {},
+    capacityThreshold: Float = 0.5f,
+    onSliderChange: (Float) -> Unit = {},
+    initialSelectedGyms: Set<String> = emptySet(),
+    onGymSelected: (Set<String>) -> Unit = {},
+    onBackClick: () -> Unit = {}
+) {
```

Comment:
I guess it's a bit awkward to have all these parameters without a VM yet. Add a TODO here to change these over to VM calls!